update/genesis/import.go

```diff
@@ -400,13 +411,13 @@ func (si *stateImport) importState(fileName string) error {
 
 	var marshalledData []byte
 	var address []byte
-	for {
-		key, marshalledData, err = si.reader.ReadNextItem(fileName)
+	for _, key := range keys {
+		marshalledData, err = si.hardforkStorer.Get(key)
 		if err != nil {
 			break
 		}
 
-		keyType, address, err = GetKeyTypeAndHash(key)
+		keyType, address, err = GetKeyTypeAndHash(string(key))
 		if err != nil {
 			break
```

Comment:
return err directly. no need for break and if after for

Reply:
I wanted to do a general error wrapping at the end of the function